src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBStreamTestRunner.java

```diff
@@ -237,7 +240,8 @@ void await(int timeoutSeconds) {
             log.info("waiting " + timeoutSeconds + "s for " + this.countDownLatch.getCount() + " records to arrive ...");
             try {
                 this.countDownLatch.await(timeoutSeconds, TimeUnit.SECONDS);
-            } catch (InterruptedException ignore) {}
+            } catch (InterruptedException ignore) {
```

Comment:
not appropriate to fix in this PR, but hoo boy does exception swallowing freak me out

Reply:
That's a fair criticism.  But I'm going to invoke the "it's only a test" defense yet again.